2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1177,7 +1180,7 @@ Il progetto deve implementare le seguenti attività in conformità con le politi
     - raccogliere, archiviare e verificare i dati:
         - i dati prodotti vengono salvati e analizzati al fine di perorare una strategia di approccio ed eventuali modifiche da effettuare.
 
-    - registrare i risultati e comunicarli agli stakeholders:
+    - registrare i risultati e comunicarli agli stakeholder:
         - vengono riferiti periodicamente al Proponente gli avanzamenti e la conformità del progetto alle richieste fatte.
 
```

Comment:
Aggiungerei una nota in fondo ove si segnala che "Maggiori dettagli in merito alla definizione, al calcolo e l'analisi delle metriche sono reperibili all'interno del documento _Piano di Qualifica_ + versione"